packages/react-app/src/components/FreaksList/freaks-list.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import '../../freaks.css';
+import PropTypes from 'prop-types';
+import useEthersContract from '../../hooks/useEthersContract';
+
+function FreaksList({ value }) {
+  const [ freaks, setFreaks ] = useState([ ]);
+  const contract = useEthersContract();
+
+  useEffect(() => {
+    function parseFreaks({ _name, _scor }) {
+      const names = _name.map((name) => ({ name }));
+
+      _scor.slice(0, names.length).forEach((scor, index) => {
+        names[index] = { ...names[index], share: scor };
+      });
+
+      return names;
+    }
+
+    function loadFreaks() {
+      contract.getFreaks()
+        .then(parseFreaks)
+        .then(setFreaks)
+        .catch((error) => {
+          console.error(error);
+        });
+    }
+
+    if (contract !== null) {
+      loadFreaks();
+    }
+  }, [ contract ]);
+
+  function renderFreaks(freak, index) {
```

Comment:
It is a single freak, so:

```suggestion
  function renderFreak(freak, index) {
```